src/main/java/robot/subsystems/shooter/commands/Shoot.java

```diff
@@ -0,0 +1,64 @@
+package robot.subsystems.shooter.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.command.Command;
+
+import static robot.Constants.Shooter.*;
+import static robot.Robot.shooter;
+
+/**
+ *
+ */
+public class Shoot extends Command {
+    private double distance;
+    private double timeout;
+    private Timer timer = new Timer();
+
+    public Shoot(double distance, double timeout) {
+        requires(shooter);
+        this.distance = distance;
+        this.timeout = timeout;
+    }
+
+    public Shoot(double distance) {
+        this(distance, 0);
+    }
+
+    // Called just before this Command runs the first time
+    @Override
+    protected void initialize() {
+        timer.reset();
+        timer.start();
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    @Override
+    protected void execute() {
+        shooter.setSpeed(calculateInitialVelocity(distance) / RADIUS);
+    }
+
+
+    private double calculateInitialVelocity(double distance) {
+        double numerator = Math.pow(distance, 2) * 10;
```

Comment:
```suggestion
        double numerator = Math.pow(distance, 2) * 10;
```
Put g in a constant for clarity. Also, use a more accurate value. You should only use `g = 10m/s²` in your physics class, where you don't need accuracy. Here, accuracy is important. Use `g = 9.80665m/s²`. This is the value of `g`` according to ISO standard.